src/databricks/labs/lsql/dashboards.py

```diff
@@ -724,6 +749,72 @@ def _get_query_widget_spec(fields: list[Field], *, frame: WidgetFrameSpec | None
         return spec
 
 
+@unique
+class FilterType(str, Enum):
+    """The filter widget type"""
+
+    DATE_RANGE_PICKER = "DATE_RANGE_PICKER"
+    MULTI_SELECT = "MULTI_SELECT"
+    NONE = "NONE"
+
+    def as_widget_spec(self):
+        widget_spec_mapping = {
+            "DATE_RANGE_PICKER": DateRangePickerSpec,
+            "MULTI_SELECT": MultiSelectSpec,
+        }
+        if self.name not in widget_spec_mapping:
+            raise ValueError(f"Can not convert to filter widget spec: {self}")
+        return widget_spec_mapping[self.name]
+
+
+@dataclass
+class FilterTile(Tile):
+    _position: Position = dataclasses.field(default_factory=lambda: Position(0, 0, 3, 1))
+    dataset_columns: list[tuple[str, str]] = dataclasses.field(default_factory=list)  # (column, dataset_name)
+
+    def validate(self) -> None:
+        """Validate the tile
+
+        Raises:
+            ValueError : If the tile is invalid.
+        """
+        super().validate()
+        if not self.metadata.is_filter():
+            raise ValueError(f"Tile is not a filter file: {self}")
+
+    def get_layouts(self) -> Iterable[Layout]:
+        """Get the layout(s) reflecting this tile in the dashboard."""
+        if not self.dataset_columns:
+            logger.warning(f"Filter tile for path {self.metadata.path} has no matching dataset columns")
+            return
+        config = json.loads(self.content)
+        filter_type = FilterType[config.get("type", "MULTI_SELECT").upper()]
```

Comment:
`FilterType` conversion from string should happen in the `FilterHandler` during parsing

``` python
        parser.add_argument(
            "--type",
            type=lambda v: WidgetType(v.upper()),
            default=WidgetType.AUTO,
            help=(
                "The widget type to use, see classes with WidgetSpec as parent class in "
                "databricks.labs.lsql.lakeview.model."
            ),
            dest="widget_type",
        )
```